### PR TITLE
Allow inverted playlist by date added

### LINKED ISSUE #7: Playlist Inversion
**Is your feature request related to a problem? Please describe.**
I have been using the playlist editor to create playlists from Spotify recommendations (e.g. I start listening to a song's radio or an album and Spotify keeps adding new tracks after they are finished). However, I'd like to listen in the same order I listened to the first time.

**Describe the solution you'd like**
I'd like an option to invert the sequence of tracks on a playlist. Since I am able to create a playlist from my recent tracks, inverting this playlist would allow me to listen in the same order I listened to it the first time.

An option for inversion here would be awesome.
image

### PR DESCRIPTION
**Pull request by k9ur**
Resolves #7, only available for inversion by date added. Logical framework for inversion by other attributes is setup however.

### CODE REVIEW

**Review by juliobguedes — APPROVED** [review]
Seems a nice solution :)

**Comment by JRobsonJr** [comment]
Thanks for the contribution, @otta8634 (and sorry for the long wait until this was merged) <3

**Pull request merged**
